.github/workflows/release.yaml

```
@@ -2,12 +2,26 @@ name: release
 on:
   release:
     types: [released]
+  workflow_call:
+    inputs:
+      dry-run:
+        description: "Whether to do a dry run"
+        required: false
+        default: true
+        type: boolean
   workflow_dispatch:
+    inputs:
+      dry-run:
+        description: "Whether to do a dry run"
+        required: false
+        default: true
+        type: boolean
 
 jobs:
   brew-dispatcher:
     name: Release on homebrew-prql
     runs-on: ubuntu-latest
+    if: github.event.inputs.dry-run != 'true'
```

nit: does this work?

```suggestion
    if: ! github.event.inputs.dry-run
```

---

I wasn't sure it would work, but as you pointed out in your comment (https://github.com/PRQL/prql/pull/3181#discussion_r1278603944), GitHub Actions seems to evaluate an empty string as false, so it could work.
I'll see.

---

It does not seem working.